extensions/amp-accordion/1.0/storybook/Basic.amp.js

```diff
@@ -24,7 +24,10 @@ export default {
   decorators: [withKnobs, withA11y, withAmp],
 
   parameters: {
-    extensions: [{name: 'amp-accordion', version: '1.0'}],
+    extensions: [
+      {name: 'amp-bind', version: '0.1'},
```

Comment:
One ask: can we move this storybook into a separate module? E.g. `Bind.amp.js`?

Reply:
Sounds good, updated!